### PR TITLE
Fix some render issues

### PR DESCRIPTION
**Pull request by zccrs**

### CODE REVIEW

File: src/server/qtquick/woutputhelper.cpp [hunk 1]
```diff
@@ -255,6 +255,14 @@ bool WOutputHelper::commit()
 {
     W_D(WOutputHelper);
     wlr_output_state state = d->state;
+    if (state.committed == 0)
+        return false;
+
+    if (state.committed != WLR_OUTPUT_STATE_LAYERS) {
+        Q_ASSERT_X(state.committed & WLR_OUTPUT_STATE_BUFFER, Q_FUNC_INFO,
+                   "WOutputHelper::commit: buffer is not set, you will got a black frame");
```
Comment (asterwyx) [inline src/server/qtquick/woutputhelper.cpp:251]: blank frame?

**Review by asterwyx — COMMENTED** [review]
/+1

File: examples/tinywl/PrimaryOutput.qml [hunk 2]
```diff
@@ -95,7 +95,7 @@ OutputItem {
 
         SequentialAnimation on rotation {
             id: ani
-            running: true
+            running: false
```
Comment (wineee) [inline examples/tinywl/PrimaryOutput.qml:98]: why?

Comment (zccrs) [reply examples/tinywl/PrimaryOutput.qml:98]: test

**Pull request closed without merging**